sceptre/nlpsandbox/config/config.yaml

```diff
@@ -0,0 +1,5 @@
+project_code: nlp-sandbox
+# profile: {{ var.profile | default("default") }}
```

Comment:
you'll want to enable this, it will allow you to pass in an aws profile to sceptre command, i.e. `sceptre --var "profile=MY-PROFILE" --var "region=us-east-1" launch  ...`

Reply:
I'm update this to use `profile: {{ var.profile | default() }}` like we did [here](https://github.com/Sage-Bionetworks-Workflows/aws-workflows-nextflow-infra/blob/prod/config/config.yaml#L2) because that's what works with the simplified CI workflow.